src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/GitRepositoryProjectInspector.java

```diff
@@ -104,14 +111,20 @@ public void run() {
                 }
             }
             // Add the next steps
-            cloneRepo
+
+            if (!RepairnatorConfig.getInstance().isStaticAnalysis()) {
```

Comment:
polymorphism?

Reply:
I moved the inspector choosing in the Main processes :) .